networking/common/network.sh

```diff
@@ -136,7 +136,7 @@ reset_network_env()
 		pkill -f "nc -l"
 	
 		rsync -a --delete $networkLib/network-scripts.no_nm/ /etc/sysconfig/network-scripts/
-		 service network restart
+		service network restart
```

Comment:
@emanchado did you need to change this to use rlServiceStop and rlServiceStart ? Also I wonder if rlServiceRestore will do the equivalent ?

Reply:
@emanchado agreed I don't think it's necessary to create a new function, just curious if it was the same, sounds like it is indeed different based on the man page, it looks like you still have `service network restart` should this be changed to rlService ?

Reply:
No, `networking/common/network.sh` is one of the cases in which the test actually checks for the RHEL version (or the existence of `systemctl`), so this code is only run for old RHEL installations. The commit has the list of such files I found, to help investigating this in the future:

https://github.com/CKI-project/tests-beaker/pull/399/commits/b3ed0fa893e1da485c5ff0fb7079bafb56d597f8

Reply:
Ah ok thanks for confirming, I see that now :-)